defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# A commit hash, tag, or branch name indicating what version of the
+# Metasploit Framework installer to use.
+installer_version: 6.2.20
```

Comment:
I would prefer always quoting numeric-y values just to reinforce that they are strings in cases like this.

```suggestion
installer_version: "6.2.20"
```